modules/ROOT/pages/6.7.2-release-notes.adoc

```diff
@@ -54,8 +54,13 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 === css loading in `toolbardrawertoggletest` flickering.
 // TINY-10249
 
-// CCFR
+Previously, when a {productname} editor instance would load the content CSS for the `toolbardrawertoggletest` It would cause a noticeable flicker to the editor's UI.
```

Comment:
Actually it's not the whole thing that's wrong, just this line.

It's unrelated to the test - that's the part I'm complaining about. It's just when the editor is reloaded.

It's a simple regression; TinyMCE 5 didn't flicker when the editor reloaded and TinyMCE 6 does.

Reply:
Updated with https://github.com/tinymce/tinymce-docs/pull/2948#discussion_r1367986706 copy edit suggestion.